src/components/clusterDetail/ResetClusterModal.tsx

```diff
@@ -25,25 +28,28 @@ type ResetClusterModalButtonProps = React.ComponentProps<typeof Button> & {
 type ResetClusterModalProps = {
   onClose: () => void;
   isOpen: boolean;
-  clusterId: Cluster['id'];
+  cluster: Cluster;
 };
 
-const ResetClusterModal: React.FC<ResetClusterModalProps> = ({ onClose, isOpen, clusterId }) => {
+const ResetClusterModal: React.FC<ResetClusterModalProps> = ({ onClose, isOpen, cluster }) => {
   const dispatch = useDispatch();
   const [isSubmitting, setIsSubmitting] = React.useState(false);
   const [error, setError] = React.useState<{ title: string; message: string } | null>(null);
+  const { addAlert } = React.useContext(AlertsContext);
+
+  const areLogsAvailable = canDownloadClusterLogs(cluster);
 
   const handleClose = () => {
     setIsSubmitting(false);
     setError(null);
     onClose();
   };
 
-  const handleSubmit = async () => {
+  const doReset = async () => {
```

Comment:
Nit: handlers are usually named 'handle...'
```suggestion
  const handleReset = async () => {
```

similar for `doDownloadLogs` -> `handleDownloadLogs`

Reply:
done